testing/postgresql-cron/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: G.J.R. Timmer <gjr.timmer@gmail.com>
+# Maintainer:
+pkgname=postgresql-cron
+pkgver=1.0.0
```

Comment:
Aports must track upstream version and that’s 1.0.0-rc.1, not 1.0.0. However, `1.0.0-rc.1` is not valid version format in APK, `pkgver` must be `1.0.0_rc1` in this case.

You can do this

```sh
pkgver=1.0.0_rc1
_pkgver=${pkgver/_rc/-rc.}
```
and use `_pkgver` inside `source` and `builddir`.